src/commands/ping.ts

```diff
@@ -5,18 +5,14 @@ const command: Command = {
 	description: "Show bot latency information.",
 	options: [],
 	hasPermission: () => true,
-	parseArguments: () => [],
-	handler: (state, message) => {
-		message
-			.reply("Pinging...")
-			.then(sent =>
-				sent.edit(
-					`<@${message.author.id}>\nWebsocket heartbeat: ${state.client.ws.ping}ms\nRoundtrip latency: ${
-						sent.createdTimestamp - message.createdTimestamp
-					}ms`
-				)
-			)
-			.catch(console.error.bind(console));
+	shouldBeEphemeral: (state, interaction) => interaction.channelID !== state.config.botChannelId,
+	handler: async (state, interaction) => {
```

Comment:
We can change this back to normal removing the async/await functionality and chain `.then`

Reply:
Yup